yarn.lock

```diff
@@ -41,44 +41,44 @@
     component-type "^1.2.1"
     join-component "^1.1.0"
 
-"@storybook/addon-actions@^3.1.8":
-  version "3.1.8"
-  resolved "https://registry.yarnpkg.com/@storybook/addon-actions/-/addon-actions-3.1.8.tgz#2b6d7aa97530b19965c1010b822f40b130ebbc4d"
+"@storybook/addon-actions@^3.2.0":
```

Comment:
Interesting, it looks like it's added as a dependency instead of a devDependency. I bet this could be moved but I might be missing something. That would be all on the reaction side - worth looking into!